### PR TITLE
Add oval and  remediation for auditd_audispd_disk_full_action

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add oval check, bash and ansible remediation for auditd_audispd_disk_full_action rule

### CODE REVIEW

**Comment by teacup-on-rockingchair** [comment]
> Can you have test scenarios for this rule?

Added those [26389dc](https://github.com/ComplianceAsCode/content/pull/9195/commits/26389dc0ea805b978c6bc94eff53112f28b6ab54) 
Since I keep forgetting that policy do you think that it worth investing time in researching the option to add a check for it in the PR automatic checks, if there is new oval rule added, make sure it is added with tests at least for pass and fail? If so which check do you think might be best to put that logic into?

File: linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_disk_full_action/ansible/shared.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
```
[the remark this answers is not in the document]
Comment (teacup-on-rockingchair) [reply linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_disk_full_action/ansible/shared.yml:1]: Thanks :bow:  do you think that I might do the same for all the other `configure_auditd_data_retention/auditd_audispd*` rules and commit that change in this PR ?

[the remark this answers is not in the document]
Comment (jan-cerny) [reply linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_disk_full_action/ansible/shared.yml:1]: You can, but I don't know if it's a valueable time spend. There is a problem is that there is a mess in these "platforms" across the whole project, so for the new rules I recommend to make them relaxed.

**Comment by jan-cerny** [comment]
That's a great idea!

@mildas Could CTF do something like that?

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**